values.yaml

```diff
@@ -1612,6 +1621,15 @@ concourse:
       ##
       reapInterval: 10s
 
+      ## 4 for IPv4 and 6 for IPv6
+      ##
+      p2pInterfaceFamily:
+
+      ## Regular expression to match a network interface for p2p streaming
+      ## e.g. "eth0"
+      ##
+      p2pInterfacePattern:
```

Comment:
Would be nice to rename this `p2pInterfaceNamePattern` to keep consistent with `CONCOURSE_BAGGAGECLAIM_P2P_INTERFACE_NAME_PATTERN`. 

Or vice-versa, rename env var to `CONCOURSE_BAGGAGECLAIM_P2P_INTERFACE_PATTERN`.

Reply:
Good call, changed it!